packages/flutter/test/widgets/routes_test.dart

```diff
@@ -1700,8 +1700,8 @@ void main() {
       final WidgetWithLocalHistoryState state = tester.state(find.byType(WidgetWithLocalHistory));
       state.addLocalHistory();
       // Waits for modal route to update its internal state;
-      await tester.pump();
 
+      await tester.pump();
```

Comment:
nit: revert this change here? This line belongs to the comment above and there shouldn't be a blank line in between.